packages/jellyfish-api-core/__tests__/category/account/listBurnHistory.test.ts

```diff
@@ -0,0 +1,261 @@
+  })
+
+  it('should listBurnHistory with filter on txtype CreateMasternode', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.CREATE_MASTERNODE
+    })
+    expect(history.every(({ type }) => type === 'CreateMasternode')).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on txtype CreateToken', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.CREATE_TOKEN
+    })
+    expect(history.every(({ type }) => type === 'CreateToken')).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on token DFI', async () => {
+    const history = await client.account.listBurnHistory({
+      token: 'DFI'
+    })
+    expect(history.every(({ amounts }) => amounts[0].includes('DFI'))).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on token GOLD', async () => {
+    const history = await client.account.listBurnHistory({
+      token: 'GOLD'
+    })
+    expect(history.every(({ amounts }) => amounts[0].includes('GOLD'))).toBeTruthy()
+  })
+
+  it('should listBurnHistory with maxBlockHeight 110', async () => {
+    const maxBlockHeight = 110
+    const history = await client.account.listBurnHistory({
+      maxBlockHeight
+    })
+    expect(history.every(({ blockHeight }) => blockHeight <= maxBlockHeight)).toBeTruthy()
+  })
+
+  it('should listBurnHistory with maxBlockHeight 500', async () => {
+    const maxBlockHeight = 500
+    const history = await client.account.listBurnHistory({
+      maxBlockHeight
+    })
+    expect(history.every(({ blockHeight }) => blockHeight <= maxBlockHeight)).toBeTruthy()
+  })
+
+  it('should listBurnHistory with depth 0', async () => {
+    const depth = 0
+    const history = await client.account.listBurnHistory()
+    const depthHistory = await client.account.listBurnHistory({
+      depth
+    })
+    const maxBlockHeight = Math.max(...history.map(el => el.blockHeight)) // Get maxBlockHeight from history
+    expect(depthHistory.every(({ blockHeight }) => blockHeight >= maxBlockHeight - depth)).toBeTruthy()
+  })
+
+  it('should listBurnHistory with depth 10', async () => {
+    const depth = 10
+    const history = await client.account.listBurnHistory()
+    const depthHistory = await client.account.listBurnHistory({
+      depth
+    })
+    const maxBlockHeight = Math.max(...history.map(el => el.blockHeight)) // Get maxBlockHeight from history
+    expect(depthHistory.every(({ blockHeight }) => blockHeight >= maxBlockHeight - depth)).toBeTruthy()
+  })
+
+  it('should listBurnHistory with limit 1', async () => {
+    const history = await client.account.listBurnHistory()
+    const historyWithLimit = await client.account.listBurnHistory({
+      limit: 1
+    })
+    expect(historyWithLimit.length).toStrictEqual(1)
+    expect(historyWithLimit[0]).toStrictEqual(history[0])
+  })
+
+  it('should listBurnHistory with limit 5', async () => {
```

Comment:
You wrote 2 `it` for` limit 1 `and` limit 5`
Unless you predict they return a different **pattern** of results, you should keep 1 only. 
I would recommend you keep `limit 5`